### PR TITLE
add new mtx commands

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: src/mtx/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -217,6 +217,31 @@ export interface IDeviceId {
...
+export interface ITimeCorrectionParameters {
+    monthTransitionSummer: number,
+    dateTransitionSummer: number,
+    hoursTransitionSummer: number,
+    hoursCorrectSummer: number,
+    monthTransitionWinter: number,
+    dateTransitionWinter: number,
+    hoursTransitionWinter: number,
+    hoursCorrectWinter: number,
+    needCorrection: boolean
```
Comment (DeadByElpy) [inline src/mtx/CommandBinaryBuffer.ts:229]: maybe `isCorrectionNeeded`?

File: src/mtx/commands/uplink/SetCorrectTimeResponse.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ * import SetCorrectTimeResponse from 'jooby-codec/mtx/commands/uplink/SetCorrectTimeResponse.js';
+ *
+ * const command = SetCorrectTimeResponse.fromBytes();
+ *
+ * console.log(command.parameters);
+ * // output:
+ * undefined
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/TurnRelayOn.md#response)
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/SetCorrectTimeResponse.ts:31]:
```suggestion
 * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/uplink/SetCorrectTimeResponse.md#response)
```

File: src/mtx/commands/uplink/GetVersionResponse.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ * const command = GetVersionResponse.fromBytes(commandBody);
+ *
+ * console.log(command.parameters);
+ * // output:
+ * {
+ *     version: '104.25.003'
+ * }
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/uplink/GetVersionResponse.md)
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/GetVersionResponse.ts:42]:
```suggestion
 * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/GetVersion.md#response)
```

?

File: src/mtx/commands/uplink/SetSaldoResponse.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ * import SetSaldoResponse from 'jooby-codec/mtx/commands/uplink/SetSaldoResponse.js';
+ *
+ * const command = SetSaldoResponse.fromBytes();
+ *
+ * console.log(command.parameters);
+ * // output:
+ * undefined
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/TurnRelayOn.md#response)
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/SetSaldoResponse.ts:31]: bad link

File: src/mtx/commands/uplink/SetSaldoParametersResponse.ts [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+ * import SetSaldoParametersResponse from 'jooby-codec/mtx/commands/uplink/SetSaldoParametersResponse.js';
+ *
+ * const command = SetSaldoParametersResponse.fromBytes();
+ *
+ * console.log(command.parameters);
+ * // output:
+ * undefined
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/TurnRelayOn.md#response)
```
[the remark this answers is not in the document]
Comment (DarkPark) [reply src/mtx/commands/uplink/SetSaldoParametersResponse.ts:31]: fixed

File: src/mtx/commands/uplink/SetSaldoResponse.ts (hunk 4 above)
Comment (DarkPark) [reply src/mtx/commands/uplink/SetSaldoResponse.ts:31]: fixed

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**